internal/params/config.go

```diff
@@ -342,7 +348,10 @@ var (
 		big.NewInt(0),        // SlotsLimitedEpoch
 		big.NewInt(1),        // CrossShardXferPrecompileEpoch
 		big.NewInt(0),        // AllowlistEpoch
-		big.NewInt(0),        // TesnetNinetyPercentEpoch
+		// TODO place correct epoch number
```

Comment:
In addition to this `TODO`, please also add `LeaderRotationEpoch` to be `EpochTBD` for all non-testnet chains.

Reply:
fixed